cumulusci/tasks/salesforce/UpdateAdminProfile.py

```diff
@@ -176,19 +183,18 @@ def _set_tabs_visibility(self):
             elem.find("sf:visibility", self.namespaces).text = "DefaultOn"
 
     def _deploy_metadata(self):
-        deploy_dir = tempfile.mkdtemp()
-        self.logger.info(f"Deploying updated Admin.profile from {deploy_dir}")
+        self.logger.info(f"Deploying updated Admin.profile from {self.deploy_dir}")
 
-        target_profile_xml = Path(deploy_dir, "package.xml")
+        target_profile_xml = Path(self.deploy_dir, "package.xml")
         target_profile_xml.write_text(
             """<?xml version="1.0" encoding="UTF-8"?><Package xmlns="http://soap.sforce.com/2006/04/metadata">
         <types><members>Admin</members><name>Profile</name></types><version>39.0</version></Package>
         """
         )
 
-        source_profile_dir = Path(self.tempdir, "profiles")
-        target_profile_dir = Path(deploy_dir, "profiles")
-        source_profile_dir.replace(target_profile_dir)
+        retreived_profile_dir = Path(self.retrieve_dir, "profiles")
+        target_profile_dir = Path(self.deploy_dir, "profiles")
+        retreived_profile_dir.replace(target_profile_dir)
```

Comment:
retrieved

I don't really understand the reason for using separate directories for retrieval and deployment. We can have a single temporary working directory where we retrieve, then overwrite package.xml and then deploy just the profile, no?

Reply:
> retrieved

Thanks. Right up there with "receipt" among my most misspelled words. 

I misread your comment above, I think. My rationale is that we shouldn't send extraneous data over the network if we can help it. I haven't done any testing of deployment times when sending extra data, so I can't say whether it's any slower. I think it's worth the extra ~20 LoC, but I'm receptive to counterarguments.

Reply:
@jstvz oh right, I hadn't considered that our deploy task is naive and will include everything in the folder even if it's not in package.xml. okay, you convinced me.

Reply:
Okay. Went back and tested this for my own edification:
```
Update of CustomField Contact.Phone: Error: Not in package.xml
```